contentpal/src/main/java/org/dmfs/android/contentpal/tools/ClosableEmptyIterator.java

```diff
@@ -29,7 +30,7 @@
  *
  * @author Marten Gajda
  */
-public final class ClosableEmptyIterator<T> extends AbstractBaseIterator<T> implements ClosableIterator<T>
+public final class ClosableEmptyIterator<T> extends AbstractBaseIterator<T> implements Iterator<T>, ClosableIterator<T>
```

Comment:
Why are these explicit `Iterator<T>` declarations needed beside extending `CloseableIterator` here and in `FakeCloseable`?

Reply:
It isn't I guess Intellij added it when I deleted the `ClosableIterator` interface. I'll remove it.

Reply:
Fixed. @lemonboston please check again.